spec/board_spec.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/board.rb'
+
+RSpec.describe 'testing the board' do
+  it 'should show us an empty board' do
+    expect(Board.new.show_board).to eq("+------------------------+\n| 1 - ❔  2 - ❔  3 - ❔ | \n| 4 - ❔  5 - ❔  6 - ❔ | \n| 7 - ❔  8 - ❔  9 - ❔ | \n+------------------------+")
+  end
+
+  it 'should show us an empty board' do
+    expect(Board.new.show_board).not_to eq("+------------------------+\n| 1 - p  2 - ❔  3 - ❔ | \n| 4 - ❔  5 - ❔  6 - ❔ | \n| 7 - ❔  8 - ❔  9 - ❔ | \n+------------------------+")
+  end
```

Comment:
These tests should be wrapped in a `describe` block. Since both of them are testing the `show_board` method, they should have their own `describe` block.

- [ ] Please wrap them in a `describe` block.